package.json

```diff
@@ -159,6 +159,7 @@
     "execa": "^6.1.0",
     "ipfs-utils": "^9.0.1",
     "joi": "^17.2.1",
+    "kubo-rpc-client": "^1.0.0",
```

Comment:
This should be a dev dep the same as `ipfs`, `ipfs-client` and `ipfs-http-client`, otherwise you won't be able to use this module to test with semver-incompatible versions of `kubo-rpc-client`.